test/git/duo/author_collection_test.rb

```diff
@@ -21,6 +21,16 @@ def test_where_partial_match
       assert_equal "Alfred Pennyworth", result.name
     end
 
+    def test_closest_partial_match
```

Comment:
@teoljungberg Would that be something along the lines of `test_where_partial_matches_closest`?